apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/Trace/SimpleManagedTracer.cs

```diff
@@ -92,27 +109,20 @@ public IDisposable StartSpan(string name, StartSpanOptions options = null)
             GaxPreconditions.CheckNotNull(name, nameof(name));
             options = options ?? StartSpanOptions.Create();
 
-            TraceSpan span = new TraceSpan();
-            span.SpanId = _spanIdFactory.NextId();
-            span.Kind = options.SpanKind.Convert();
-            span.Name = name;
-            span.StartTime = Timestamp.FromDateTime(DateTime.Now.ToUniversalTime());
-
-            lock (_stackLock)
+            var currentStack = TraceStack;
+            var span = new TraceSpan
             {
-                if (_traceStack.Count != 0)
-                {
-                    span.ParentSpanId = _traceStack.Peek().SpanId;
-                }
-                else if (_rootSpanParentId != null)
-                {
-                    span.ParentSpanId = (ulong)_rootSpanParentId;
-                }
+                SpanId = _spanIdFactory.NextId(),
+                Kind = options.SpanKind.Convert(),
+                Name = name,
+                StartTime = Timestamp.FromDateTime(DateTime.Now.ToUniversalTime()),
```

Comment:
Use `DateTime.UtcNow`. I realize this is an extra change to the previous code, but we really shouldn't have DateTime.Now anywhere :)